.github/workflows/screenshots.yml

```diff
@@ -18,7 +18,14 @@ jobs:
       PERCY_TOKEN: ${{ secrets.PERCY_TOKEN }}
       PORT: 8888
 
+    # Skip when secrets are unavailable on forks
+    if: ${{ !github.event.pull_request.head.repo.fork }}
+
     steps:
+      - name: Check secrets
+        if: ${{ !env.PERCY_TOKEN }}
+        run: echo "::warning title=GitHub Actions secrets::Workflow requires 'PERCY_TOKEN' secret"
```

Comment:
For more advanced contributors that might run the GitHub Actions workflows manually (on their fork) we add a warning message alongside the error output to explain why the Percy run failed:

<img width="356" alt="Percy secret warning" src="https://user-images.githubusercontent.com/415517/208912925-afd5ddcc-1c9c-46f4-be36-b957884ebbb3.png">